src/main/scala/io/snyk/plugin/ui/state/Navigator.scala

```diff
@@ -60,10 +61,11 @@ object Navigator extends IntellijLogging {
       * Use MavenProjectsManager to open the editor and highlight where the specified artifact
       * is imported.
       */
-    override def navigateToDependency(
-      group: String,
-      name: String,
-      projectId: String
+    override def navigateToDependency(group: String,
```

Comment:
i prefer the previous indentation

Reply:
I guess it's auto-formatting too depends on whether first parameter is in the line or not (IntelliJ: Method call arguments)